_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Mesh Expansion
+overview: Instructions to add external machines and expanding Istio.
```

Comment:
expand Istio

Reply:
would "and expand the Istio mesh." work ? (or "expand the Istio Mesh." ? do we capitalize Istio Mesh ?)

Reply:
Oh good thought--yes the first one (not capitalized): "and expand the Istio mesh."